lib/src/utils/http_helper.dart

```diff
@@ -198,14 +231,30 @@ class HttpHelper {
     return completer.future;
   }
 
+  /// Returns the request headers.
+  ///
+  /// Note: [addCredentialsToHeader] and [isTestModeActive] exclude each other.
+  Map<String, String>? _getBearerHeader(
+    final String bearerToken,
+  ) =>
+      <String, String>{
+        'Authorization': 'bearer $bearerToken',
+        'Content-Type': 'application/json',
+      };
+
   /// Returns the request headers.
   ///
   /// Note: [addCredentialsToHeader] and [isTestModeActive] exclude each other.
   Map<String, String>? _buildHeaders({
     User? user,
     required final UriHelper uriHelper,
     required bool addCredentialsToHeader,
+    final String? bearerToken,
   }) {
+    if (bearerToken != null) {
+      return _getBearerHeader(bearerToken);
+    }
```

Comment:
Don't we want a user agent and a "From" header. Just because prices does not use them yet it could be interesting

Reply:
> Don't we want a user agent and a "From" header. Just because prices does not use them yet it could be interesting

Could be done later if needed, for stats I guess.
I don't know if `prices` plan to do something with it.
For the moment the code is clearer to read that way: "token? just return this! else do as usual".